### DISCUSSION TITLE
Initial theme and layout IDEA !

### DISCUSSION THREAD

**Discussion by Vidya4sure** [virtual-tech-school/frytheweb#6] [General]
https://realtimecolors.com/?colors=48d6b5-101727-43c3d8-1e6c97-ffb084

HAVE A LOOK !

**Comment by SaarthakMaini** [comment]
I think the combination of #1e6c97 and #48d6b5 can be improved, but otherwise it's good from my perspective

**Reply by Vidya4sure** [reply]
suggest your combination. what is your taste regarding this?
